### PR TITLE
[Review] Request from 'digitaltom' @ 'SUSE/connect/review_171207_fix_suseconnect_build_for_sle15'

### PR DESCRIPTION
**Pull request by digitaltom**
Please review the following changes:
  * 6af6dcd bump version
  * 7984e07 v0.3.3
  * a045acc fix build for SLE15
  * 34146e0 git ignore generated files
  * 07c1cf3 remove extra .gitignore
  * ff1a51b update readme
  * 4c8e80b move obs package to package/, do not put .osc files in git
  * f238d0d remove package_ibs dir

### CODE REVIEW

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/SCC-Connect-Prophet/70451/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project connect on step docker-run-tests.
( Failure: Merged 6af6dcd75e22e496c33f2d6013ff0400555b8f9d into 5f35174de4e271df2bcfe81f1fbcd3dedd92d40e )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/70455/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged eada7502649c1ca4982a7286b44d37291c38b72f into 5f35174de4e271df2bcfe81f1fbcd3dedd92d40e )

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/70456/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 6b815112c6b94854cf2819aa6019da7ea88dbe80 into 5f35174de4e271df2bcfe81f1fbcd3dedd92d40e )

File: PACKAGE.md [hunk 1]
```diff
@@ -48,30 +43,25 @@ Then build the gem and copy it to package-building directory:
...
-cd ../package_obs
+cd package
 ronn --roff --manual SUSEConnect --pipe ../SUSEConnect.8.ronn > SUSEConnect.8  && gzip -f SUSEConnect.8
 ronn --roff --manual SUSEConnect --pipe ../SUSEConnect.5.ronn > SUSEConnect.5  && gzip -f SUSEConnect.5
 ```
 
 ## Step 5. Build the package
 
-Build the package for SLES 12 versions:
+Build the package for supported SLES versions:
```
Comment (lagartoflojo) [inline PACKAGE.md]: well, to be precise, openSUSE is not SLES

File: PACKAGE.md [hunk 2]
```diff
@@ -16,27 +16,22 @@ This also commits the version change. So `git show HEAD` shows you the new versi
 
 ## Step 2. Refresh your package
```
[the remark this answers is not in the document]
Comment (digitaltom) [reply PACKAGE.md]: Done.

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/70694/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 3b2aceea0f3f02e8f36d01d202e3f61e12590a00 into 5f35174de4e271df2bcfe81f1fbcd3dedd92d40e )

**Comment by digitaltom** [comment]
@lagartoflojo more improvements

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/70704/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 8fc3b608e95a597fa035c8c7a5a9a8227bb23fb1 into 5f35174de4e271df2bcfe81f1fbcd3dedd92d40e )

File: README.md [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
 is a command line tool for connecting a client system to the SUSE Customer Center.
 It will connect the system to your product subscriptions and enable the product repositories/services locally.
 
+SUSEConnect is distributed as RPM for all SUSE distributions and gets built in
+the [openSUSE build service](https://build.opensuse.org/package/show/systemsmanagement:SCC/SUSEConnect).
+
 Please visit https://scc.suse.com to see and manage your subscriptions.
 
-# Rake tasks
+SUSEConnect is communicating with SCC over this [REST API](https://github.com/SUSE/connect/blob/master/doc/SCC-API-%28Implemented%29.md).
```
Comment (lagartoflojo) [inline README.md]: SUSEConnect **communicates** with

File: README.md [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
 
-NOTE: If you are using RVM/Chruby/rbenv - make sure you installed gem2rpm as the package to the system itself and
-using it for `rake build`
+# Rake tasks
 
 ```
-rake build    # build locally (prepare for pushing to ibs)
+rake build    # build locally (see PACKAGE.md for details)
```
Comment (lagartoflojo) [inline README.md]: `rake build` requires a product parameter now 😅

File: README.md [hunk 5]
```diff
@@ -1,21 +1,24 @@
 [![Build Status](https://secure.travis-ci.org/SUSE/connect.png?branch=master)](https://travis-ci.org/SUSE/connect)
+[![Dependency Status](https://gemnasium.com/SUSE/connect.svg)](https://gemnasium.com/SUSE/connect)
 [![Code Climate](https://codeclimate.com/github/SUSE/connect.png)](https://codeclimate.com/github/SUSE/connect)
 [![Coverage Status](https://coveralls.io/repos/SUSE/connect/badge.png?branch=master)](https://coveralls.io/r/SUSE/connect)
 
-#SUSEConnect
+# SUSEConnect
 
 is a command line tool for connecting a client system to the SUSE Customer Center.
```
Comment (lagartoflojo) [inline README.md]: > is a command line tool...

I would change it to:

> SUSEConnect is a command line tool...

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/70723/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
( Success: Merged 604580a4ac97fd8b8fe1cef23e6b57e262aaaef5 into 5f35174de4e271df2bcfe81f1fbcd3dedd92d40e )

**Pull request merged**